Resources/Prototypes/GameRules/events.yml

```
@@ -545,3 +546,28 @@
     maxOccurrences: 1 # this event has diminishing returns on interesting-ness, so we cap it
     weight: 5
   - type: MobReplacementRule
+
+- type: entity
+  parent: BaseGameRule
+  id: DerelictCyborgSpawn
+  components:
+  - type: StationEvent
+    weight: 8
```

```suggestion
    weight: 5
```
I would say make it a little rarer, same as closet skeletons. Given that they have all access they will probably cause quite a lot of chaos.